examples/restart/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Globalization;
+using System.Text.Json;
+using Json.Patch;
+using k8s;
+using k8s.Models;
+
+double ConvertToUnixTimestamp(DateTime date)
+{
+    var origin = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+    var diff = date.ToUniversalTime() - origin;
+    return Math.Floor(diff.TotalSeconds);
+}
+
+async Task RestartDamonsetAsync(string name,string @namespace)
```

Comment:
nit space between the command and the `string` declaration (here and everywhere)

Reply:
Also, please add explicit disables so that this doesn't trigger a style warning (or fix the style warning). Here and below.